### PR TITLE
Update `DESCRIPTION` for CRAN submission

### PR DESCRIPTION
**Pull request by IndrajeetPatil**
Related to #113

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -30,10 +30,12 @@ Authors@R:
...
 Description: The 'easystats' is a meta-package that contains a set of
-    packages, all of which aim to provide a unifying and consistent
-    framework to tame, discipline and harness the scary R statistics and
-    their pesky models. This package is designed to make it easy to
-    install and load multiple 'easystats' packages in a single step.
+    packages, all of which provide a unifying and consistent framework for 
+    statistical modeling, visualization, and reporting. This package installs 
+    and loads multiple 'easystats' packages in a single step. Additionally, 
+    it provides articles targeted at instructors for teaching 'easystats',
+    and a dashboard targeted at new R users for accessing key 'easystats'
```
[the remark this answers is not in the document]
Comment (IndrajeetPatil) [reply DESCRIPTION]: Thanks. Check if the revised version is any better.

File: DESCRIPTION [hunk 2]
```diff
@@ -30,10 +30,12 @@ Authors@R:
              comment = c(ORCID = "0000-0001-9560-6336", Twitter = "@bmwiernik")))
 Maintainer: Dominique Makowski <dom.makowski@gmail.com>
 Description: The 'easystats' is a meta-package that contains a set of
```
Comment (bwiernik) [inline DESCRIPTION:32]: Either delete "The" or add "package" after "'easystsats'"

File: DESCRIPTION [hunk 3]
```diff
@@ -30,10 +30,12 @@ Authors@R:
...
-    packages, all of which aim to provide a unifying and consistent
-    framework to tame, discipline and harness the scary R statistics and
-    their pesky models. This package is designed to make it easy to
-    install and load multiple 'easystats' packages in a single step.
+    packages, all of which provide a unifying and consistent framework for 
+    statistical modeling, visualization, and reporting. This package installs 
+    and loads multiple 'easystats' packages in a single step. Additionally, 
+    it provides articles targeted at instructors for teaching 'easystats',
+    and a dashboard targeted at new R users for easily conducting statistical analysis 
+    by accessing summary results, model fit indices, and visualizations.
```
Comment (bwiernik) [inline DESCRIPTION:38]:
```suggestion
    by accessing summary results, model fit indices, and visualizations with minimal programming.
```

Comment (bwiernik) [reply DESCRIPTION:38]: A possibility

File: DESCRIPTION (hunk 2 above)
Comment (IndrajeetPatil) [reply DESCRIPTION:32]: Added "package" after "'easystsats'"

File: DESCRIPTION (hunk 3 above)
Comment (IndrajeetPatil) [reply DESCRIPTION:38]: Good idea. Added it.

**Pull request merged**